packages/webapp/src/components/Table/TableV2.jsx

```diff
@@ -228,7 +228,11 @@ export default function TableV2(props) {
                     return (
                       <TableCell
                         key={id}
-                        className={clsx(styles.tableCell, dense && styles.dense)}
+                        className={clsx(
+                          styles.tableCell,
+                          dense && styles.dense,
+                          shouldShowCheckbox && id === 'identification' && styles.afterCheckbox,
```

Comment:
It would be ideal if we could avoid bringing an animal specific column...
Is it possible to add the style to `animalsColumns` in Inventory container like we do for the arrow?
```
{
  id: showLinks ? 'path' : null,
  label: '',
  format: (d: AnimalInventoryItem) => (
    <Cell kind={CellKind.RIGHT_CHEVRON_LINK} path={d.path} />
  ),
  columnProps: {
    style: { width: '40px', padding: `0 ${isDesktop ? 12 : 8}px` },
  },
  sortable: false,
},
```
I hope the container knows if it needs to show checkboxes or not 🙏

Reply:
@SayakaOno could you open a PR to demo?

Reply:
Sorry, it can't be done with columnProps!! I realized it after looking at the code carefully...
Instead, I tried adding a className to the cell. Here's the test code: https://github.com/LiteFarmOrg/LiteFarm/commit/a889d3d868d07b687a74602614cab35e24b1ccfd

Reply:
That's a much better solution and actually it probably would have saved me so prop passing to do that earlier, for removed! ❤️